renderer.js

```diff
@@ -69,18 +69,28 @@ function initFloor (scene) {
 
 function addToScene (scene, node) {
   var geometry;
+  console.log("---------------------");
+  console.log(node.attributes);
   if (node.tagName === 'group') {
     return createGroup(scene, node);
   } else if (node.tagName === 'cube') {
     geometry = new THREE.BoxGeometry(1, 1, 1);
   } else if (node.tagName === 'sphere') {
     geometry = new THREE.SphereBufferGeometry(0.5);
+  } else if (node.tagName === 'cylinder') {
+    geometry = new THREE.CylinderGeometry(1, 1, 1, 32);
+  } else if (node.tagName === 'cone') {
+    geometry = new THREE.CylinderGeometry(0, 1, 2, 32);
+  } else if (node.tagName === 'pyramid') {
+    geometry = new THREE.CylinderGeometry(0, 1, 1, 4, 1);
   } else if (node.tagName === 'scene') {
     return;
   } else {
     throw new Error('unrecognized type');
   }
-  var material = new THREE.MeshLambertMaterial( { color: 0x00ff00 } );
+  var colorAttr = node.attributes.getNamedItem('color');
```

Comment:
this is awesome! :+1: so people know about it, it should probably be documented somewhere - like in the `README`.